test/index.ts

```diff
@@ -1,22 +1,52 @@
-global['fetch'] = require('cross-fetch');
+// const { JsonRpcProvider } = require('@ethersproject/providers');
```

Comment:
This file shouldn't be changed, and you should create a file "examples.json" using this format https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/erc20-balance-of/examples.json with a sample of request for your strategies that will give positive score.